src/GitVersion.Core/Git/SemanticVersion.cs

```diff
@@ -225,39 +217,29 @@ private static bool TryParseLoose(string version, [NotNullWhen(true)] out Semant
     public int CompareTo(SemanticVersion? value, bool includePrerelease)
     {
         if (value == null)
-        {
             return 1;
-        }
-        if (this.Major != value.Major)
+        if (Major != value.Major)
         {
-            if (this.Major > value.Major)
-            {
```

Comment:
I don't agree with the removal of these curly braces. Please add them back and adjust the formatting settings accordingly.

Reply:
Really strange I didn't change it explicit by myselfe. It might be because of some tooling on my side. sorry for that I change it back

Reply:
I have changed it back